shopify/product-tagger-public-remix-ssr/web/routes/_app.tsx

```diff
@@ -0,0 +1,49 @@
+import { useLoaderData, Outlet } from "@remix-run/react";
+import { json, type LoaderFunctionArgs } from "@remix-run/node";
+import { Page, Card, Text, Box } from "@shopify/polaris";
+import { NavMenu } from "../components/NavMenu";
+
+export const loader = async ({ context }: LoaderFunctionArgs) => {
+  return json({
+    gadgetConfig: context.gadgetConfig,
+  });
+};
+
+export default function () {
+  const { gadgetConfig } = useLoaderData<typeof loader>();
+
+  return gadgetConfig.shopifyInstallState ? (
+    <>
+      <NavMenu />
+      <Outlet />
+    </>
+  ) : (
+    <Unauthenticated />
+  );
+}
+
+const Unauthenticated = () => {
+  const { gadgetConfig } = useLoaderData<typeof loader>();
+
+  return (
+    <Page>
+      <div style={{ height: "80px" }}>
+        <Card padding="500">
+          <Text variant="headingLg" as="h1">
+            App must be viewed in the Shopify Admin
+          </Text>
+          <Box paddingBlockStart="200">
+            <Text variant="bodyLg" as="p">
+              Edit this page:{" "}
+              <a
+                href={`/edit/${gadgetConfig.environment}/files/web/components/App.tsx`}
+              >
+                web/components/App.tsx
+              </a>
+            </Text>
```

Comment:
I've just found an error in our default Shopify Remix SSR code. It should be:
```suggestion
            <Text variant="bodyLg" as="p">
              Edit this page:{" "}
              <a
                href={`/edit/${gadgetConfig.environment}/files/web/components/_app.tsx`}
              >
                web/components/_app.tsx
              </a>
            </Text>
```
I'm updating our default templates.

Reply:
I think that's still wrong. Should it not be routes rather than components?

Reply:
Ahhhh right

Reply:
Updated the suggestion for your reference.